app/public/style.css

```diff
@@ -0,0 +1,138 @@
+
+  .wrapper {
+    display: flex;
+    flex-flow: row wrap;
+  }
+
+  /* We tell all items to be 100% width */
+  .header, .main, .nav, .aside, .footer {
+    flex: 1 100%;
+
+  }
+
+  /* We rely on source order for mobile-first approach
+   * in this case:
+   * 1. header
+   * 2. nav
+   * 3. main
+   * 4. aside
+   * 5. footer
+   */
+
+  /* Medium screens */
+  @media all and (min-width: 600px) {
+    /* We tell both sidebars to share a row */
+    .aside { flex: 1 auto; }
+  }
+
+  /* Large screens */
+  @media all and (min-width: 800px) {
+    /* We invert order of first sidebar and main
+     * And tell the main element to take twice as much width as the other two sidebars
+     */
+    .main { flex: 2 0px; }
+
+    .aside-1 { order: 1; }
+    .main    { order: 2; }
+    .aside-2 { order: 3; }
+    .footer  { order: 4; }
+  }
+
+  @import "compass/css3";
+
+.wrapper {
+  display: -webkit-box;
+  display: -moz-box;
+  display: -ms-flexbox;
+  display: -webkit-flex;
+  display: flex;
+
+  -webkit-flex-flow: row wrap;
+  flex-flow: row wrap;
+
+  font-weight: bold;
+  text-align: center;
+}
+
+.wrapper > * {
+  /*padding: 10px;*/
+  flex: 1 100%;
+}
+
+* {
+  color: #FCF6DC;
+  background-color: #000A16;
+  font-family: Circular,helvetica;
+}
+
+.header {
+  margin: auto;
+  /*width: 45%;*/
```

Comment:
Avoid committing commented code - especially for CSS, where I don't have a clue what this means :fried_shrimp: